README.md

```diff
@@ -251,6 +252,50 @@ snakemake \
 After successful execution of the workflow, results and logs will be found in
 the `results/` and `logs/` directories, respectively.
 
+### Expected output files
+
+Upon successful execution of _MIRFLOWZ_, the tool automatically removes all
+intermediate files generated during the process. The final output comprises:
+
+1. A SAM file containing alignments intersecting a pri-miR locus. These
+alignments intersect with extended start and/or end positions specified in the
+provided pri-miR annotations. Please note that they may not contribute to the
+final counting and will not appear in the final table.
+
+2. A SAM file containing alignments intersecting a miRNA locus. Similar to the
+previous file, these alignments intersect with extended start and/or end
+positions specified in the provided miRNA annotations. They may not contribute
+to the final counting and might be absent from the final table.
+
+3. A SAM file containing the uncollapsed set of alignments that contribute to
+the final counting.
+
+4. A BAM file containing the uncollapsed set of alignments contributing to the
+final counting and its corresponding index file (`bam..bai`).
+
+5. Table(s) containing the counting data from all libraries for (iso)miRs
+and/or pri-miRs. Each row corresponds to a miRNA species, and each column
+represents a sample library. Counting involves aggregating contributions from
+all alignments, calculated as the ratio of collapsed reads in th alignment to
+the number of hits (NH value).
```

Comment:
I think we take care of collapsing, so I don't think you need to mention it here. Rather, speak from the perspective of a single read. Also, the sentence is hard to understand and contains a typo. Better to say something like:

> Each read is counted towards all the miRNA species it aligns to, with 1 / n, where n is the number of genomic or transcrptomic loci that the read aligns to.

I'm not actually this is correct (is it genomic _and_ transcriptomic? do we really consider _all_ such loci, or only those for which miRs are annotated?), but I'm sure you will know it better and can adapt accordingly.